src/main/java/org/twentyfour092023/Calculator.java

```diff
@@ -0,0 +1,40 @@
+package org.twentyfour092023;
+
+public class Calculator implements ICalculator {
+
+    public int a;
+    public int b;
+    int result;
+    @Override
+    public int add(int a, int b) {
+
+        result = a+b ;
+        return result ;
+    }
+
+    @Override
+    public int subtract(int a, int b) {
+        result =a-b;
+        return result;
+
+    }
+
+    @Override
+    public int multiply(int a, int b) {
+        result =a*b;
+        return result;
+
+    }
+
+    @Override
+    public int divide(int a, int b) {
+        result =a/b;
+        return result;
```

Comment:
Learn exception handling, and catch a/0 scenario arithmetic exception.  
return a/b